lib/gollum/views/search.rb

```diff
@@ -15,6 +15,7 @@ def results
           end
           [a[:filename_count], a[:count]] <=> [b_filename_count, b_count]
         end.reverse.slice((@page_num - 1) * @max_count, @max_count)
+        sorted.each {|x| x[:href] = page_route(x[:name])}
```

Comment:
Can't we add the `:href` attribute within the `sort` block above (e.g. immediately after line 14)? Not sure that would work, but if it does, it would be more efficient

Reply:
I don't think so. Each element in an array can be referenced many times in the `sort` block.